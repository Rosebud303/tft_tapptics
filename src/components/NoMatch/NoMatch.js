import React from 'react';
import { Link } from 'react-router-dom';
import sadMumu from '../../assets/images/sad_mumu.png';

export default function NoMatch() {
  return (
    <div>
      <img src={sadMumu} alt="amumu, the sad mummuy. sorry, this page does not exist" />
      <h2>Sorry, that page does not exist!</h2>
      <Link to="/">
        <button>
          Back
        </button>
      </Link>
    </div>
  )
}
