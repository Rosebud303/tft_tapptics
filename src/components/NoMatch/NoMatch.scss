.NoMatch {
  background-color: $dark-bg;
  display: grid;
  grid-template-areas: "back img";
  grid-template-columns: 50% 50%;

  img {
    grid-area: img;
    margin: auto;
  }

  div {
    color: $body-font-color;
    grid-area: back;
    margin: auto;
    text-align: center;

    button {
      background-color: $btn-bg;
      border: 1px solid $btn-border;
      border-radius: 3px;
      color: $btn-border;

      &:hover {
        cursor: pointer;
        background-color: $btn-border;
        color: $btn-bg;
      }
    }
  }
}