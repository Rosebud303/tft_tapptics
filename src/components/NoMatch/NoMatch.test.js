import React from 'react';
import NoMatch from './NoMatch';
import { shallow } from 'enzyme';

describe('NoMatch', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NoMatch />)
  });

  it('should match the snapshot with the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});