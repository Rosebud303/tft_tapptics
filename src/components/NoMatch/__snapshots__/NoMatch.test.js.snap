// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match the snapshot with the data passed in 1`] = `
<div
  className="NoMatch"
>
  <img
    alt="amumu, the sad mummuy. sorry, this page does not exist"
    src="404_amumu.png"
  />
  <div>
    <h2>
      Sorry, that page does not exist!
    </h2>
    <Link
      to="/"
    >
      <button>
        Back
      </button>
    </Link>
  </div>
</div>
`;
